stable/osg-frontier-squid/osg-frontier-squid/templates/deployment.yaml

```diff
@@ -37,7 +38,7 @@ spec:
       containers:
       # Container for the primary application, OSG Frontier Squid
       - name: osg-frontier-squid
-        image: opensciencegrid/frontier-squid:20210106-2231
+        image: opensciencegrid/frontier-squid:fresh
```

Comment:
I know @LincolnBryant had some issues with the certain tags of `frontier-squid`, do we actually want to make this change? It also seems unrelated to adding an instance ID label